### PR TITLE
Improve the Readme

### PR DESCRIPTION
**Pull request by lacymorrow**
This is my attempt at making the readme/docs more digestible to new users.

### CODE REVIEW

**Comment by lacymorrow** [comment]
cc @pvrobays

File: README.md [hunk 1]
```diff
@@ -1,336 +1,153 @@
...
 
-```
-const electron = require('electron');
-const app = electron.app;
-const path = require('path');
-const os = require('os');
+
+```js
 const ElectronPreferences = require('electron-preferences');
-// import ElectronPreferences from 'electron-preferences' //Or if you prefer to use module imports
```
Comment (pvrobays) [inline README.md:50]: In the past I've had questions on how to import it using modules. Thats why I would leave this commented in:
```
// import ElectronPreferences from 'electron-preferences' //Or if you prefer to use module imports
```

**Review by pvrobays — CHANGES_REQUESTED** [review]
Nice! I really love the adaptations. Especially the `Features` and `Field Types` sections.

File: README.md (hunk 1 above)
Comment (lacymorrow) [reply README.md:50]: Sounds good! Added it back

**Pull request merged**